recipe/meta.yaml

```diff
@@ -82,7 +81,7 @@ requirements:
     - giflib 5.1.*  # [not win]
     - json-c  # [not win]
     - freexl
-    - openjpeg
+    - openjpeg >=2.1,<2.3.0
```

Comment:
This version needs to be built wtih `openjpeg 2.1` otherwise it could not find the header, but seems to work fine with later version. Maybe we should make this pin official...